recipes-ni/ni-grpc-device/ni-grpc-device_1.0.1.bb

```diff
@@ -6,12 +6,12 @@ LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
```

Comment:
Fyi, typo in the commit header, if you want to fix it :).

Reply:
Eugh. I'll fix it before I merge. Thanks.